requirements-server.txt

```diff
@@ -11,7 +11,7 @@ boto3==1.15.7
 azure-batch==9.0.0
 
 # For testing
-mock==3.0.5
+mock==2.0.0
```

Comment:
Why do we need to downgrade mock? Was this causing dependency conflict issues?

Reply:
```
apache-beam 2.25.0 requires mock<3.0.0,>=1.0.1, but you'll have mock 3.0.5 which is incompatible.
```